packages/testing-kit/src/environment/configuration.ts

```diff
@@ -1,117 +1,61 @@
-import { Network, NetworkGuard, getNetwork } from "@marlowe.io/runtime-core";
-import { formatValidationErrors } from "jsonbigint-io-ts-reporters";
-import { unsafeEither } from "@marlowe.io/adapter/fp-ts";
-import * as E from "fp-ts/lib/Either.js";
-import { Blockfrost } from "lucid-cardano";
-import { SeedPhrase, seedPhrase } from "../wallet/seedPhrase.js";
-import {
-  RestClient,
-  mkRestClient,
-  RuntimeVersion,
-} from "@marlowe.io/runtime-rest-client";
-import * as Lucid from "lucid-cardano";
-import * as G from "@marlowe.io/runtime-rest-client/guards";
-import { MarloweJSON } from "@marlowe.io/adapter/codec";
-
-/**
- * Test Configuration : Read from an env file, it contains :
- * 1. The necessary configuration to run e2e tests over a Lucid Wallet and a Runtime.
- * 2. A Bank Wallet Seed Phrase to provision ephemeral Test Wallets
- */
-export type TestConfiguration = {
-  bank: { seedPhrase: SeedPhrase };
-  lucid: { blockfrost: Blockfrost; node: { network: Lucid.Network } };
-  runtime: {
-    version: RuntimeVersion;
-    url: URL;
-    client: RestClient;
-    node: { network: Network };
-  };
-};
-
-/**
- * Read Test Configurations from an env file
- * @returns
- */
-export const readEnvConfigurationFile =
-  async (): Promise<TestConfiguration> => {
-    const runtimeURL = readEnvRuntimeURL();
-    const lucidNetwork = readEnvLucidNodeNetwork();
-    const runtimeClient = mkRestClient(runtimeURL.toString());
-    const status = await runtimeClient.healthcheck();
-    const runtimeNodeNetwork = getNetwork(status.networkId);
-
-    if (!G.CompatibleRuntimeVersion.is(status.version)) {
```

Comment:
Where do you provide information on this ?

Reply:
In the e2e test I don't